fennel.lua

```diff
@@ -2194,6 +2245,11 @@ module.makeSearcher = function(options)
    end
 end
 
+-- Add metadata and docstrings to fennel module
+module.makeMetadata = makeMetadata
```

Comment:
When I first put it together, I had some thought about making it possible for folks to have the metadata live elsewhere. That's probably overcomplicating it, though, particularly since I didn't provide a means of changing where it's automatically stored in function definitions.

I'll remove it now - we can always make it more configurable in future if we find a good usecase for that.

Reply:
Removed!